test/online.test.js

```diff
@@ -1,123 +1,66 @@
 var assert  = require('assert');
-var Stream  = require('../storm/resources/ledgerStream');
 var Parser  = require('../lib/ledgerParser');
 var Rest    = require('../lib/hbase/hbase-rest');
+var HBase   = require('../lib/hbase/hbase-client');
 var Promise = require('bluebird');
 var fs      = require('fs');
+var PREFIX  = 'TEST_'
```

Comment:
We could change the prefix to TEST_<name>_ (where <name> is a unique name) to make sure 2 people can run the tests concurrently (since it's hitting the same hbase).